atlas_admin/templates/instances/sitestats.html

```diff
@@ -1,26 +1,21 @@
 {% extends "base.html" %}
 
-{% block title %}{%- if themeName -%}{{ themeName }}{%-  else -%}All{% endif %}
+{% block title %}{%- if themeName -%}{{ themeName }}{%- elif nodeCount -%}{{ nodeCount }}{%- elif nodeType -%}{{ nodeType }}{%-  else -%}All{% endif %}
+
 instances{% endblock %}
 
 {% block content %}
 {%- if instanceList -%}
 <div class="row">
     <div class="columns twelve">
         <table>
-            <tr>
-                <th>Name</th>
-                <th>Site Owners</th>
+          <tr>
+            <th width="75%">Name and Site Owner</th>
             </tr>
             {% for (i, j, k) in instanceList %}
             <tr>
-                <td><a href="{{ url_for('.index') }}instances/id/{{ i }}">{{ j }}</a> </td>
-                <td>
-                    <ul>
-                        {% for person in k %}
-                        <li> {{ person }} </li>
-                        {% endfor %}
-                    </ul>
+                <td><a href="{{ url_for('.index') }}instances/{{ i }}">{{ j }}</a> <br />
```

Comment:
404 Error on links due to incorrect path (missing id), should be: 
`<td><a href="{{ url_for('.index') }}instances/id/{{ i }}">{{ j }}</a> <br />`

Reply:
Changes made; please review again